src/memex/views.py

```diff
@@ -20,6 +20,7 @@
 from pyramid import security
 from pyramid.view import view_config
 from sqlalchemy.orm import subqueryload
+import venusian
```

Comment:
We need to add this to the direct dependencies in `setup.py` if we're using it directly. Because if for whatever reason pyramid will remove this dependency, then our code will break.